packages/cardpay-cli/reward-pool.ts

```diff
@@ -13,24 +13,16 @@ export async function rewardTokenBalances(
 ): Promise<void> {
   let web3 = await getWeb3(network, mnemonic);
   let rewardPool = await getSDK('RewardPool', web3);
-  if (rewardProgramId) {
-    const tokenBalances = await rewardPool.rewardTokenBalances(address, rewardProgramId);
-    console.log('\n');
-    console.log(`Reward balances for ${address}`);
-    console.log('---------------------------------------------------------------------');
-    displayRewardTokenBalance(tokenBalances);
-  } else {
-    const tokenBalances = await rewardPool.rewardTokenBalances(address);
-    console.log('\n');
-    console.log(`Reward balances for ${address}`);
-    console.log('---------------------------------------------------------------------');
-    displayRewardTokenBalance(tokenBalances);
-  }
+  const tokenBalances = await rewardPool.rewardTokenBalances(address, rewardProgramId);
+  console.log('\n');
+  console.log(`Reward balances for ${address}`);
```

Comment:
anytime you have `console.log('\n');` that probably means you should just use `s like this:
```js
console.log(`
Reward balances for ${address}`)'
```